src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media_Animation/DoubleAnimationTestsControl.xaml.cs

```diff
@@ -213,8 +213,7 @@ void handleSelection(object sender, object args)
 
 		private Action Dispatch(Func<CancellationToken, Task> action) => () =>
 		{
-#pragma warning disable CS4014 // Async call not awaited : Safe here (try/catch)
-			Dispatcher.RunAsync(CoreDispatcherPriority.Normal, async () =>
+			_ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, async () =>
```

Comment:
Looks like assigning the returned `IAsyncAction` to a discard makes the warning go away.

FYI @MartinZikmund @jeromelaban